tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -65,7 +65,7 @@ define(`SMART_RX_CHANNELS', 4)
 define(`SMART_FB_CHANNELS', 4)
 # Ref capture related
 define(`SMART_REF_PPL_ID', 4)
-define(`SMART_REF_CH_NUM', 4)
+define(`SMART_REF_CH_NUM', 2)
```

Comment:
a longer commit message would be appreciated, what exactly is this 'ref_ch_num'. You have 2 amps, is this one echo reference channel per amp?

Reply:
Yes, that's my understanding based on the discussion of https://github.com/thesofproject/sof/issues/3409.
As a result, voltage data only appears on 2xch not 4, will update the commit message.